client-ios/components/addScreen.js

```diff
@@ -26,6 +26,7 @@ export default class AddScreen extends Component {
     return (
       <View style={styles.container}>
         <Form />
+        <MainButtons />
```

Comment:
Since we have MainButtons component here, we don't need MainButton Scene...(FMO)